cmd/applyconfig/applyconfig.go

```diff
@@ -50,6 +52,9 @@ const (
 	dryAuto   dryRunMethod = "auto"
 	dryServer dryRunMethod = "server"
 	dryClient dryRunMethod = "client"
+
+	applyServer applyMethod = "server"
+	applyClient applyMethod = "client"
```

Comment:
Why do we need it?

Reply:
In case we switch to server-side apply globally.